package.json

```diff
@@ -48,6 +48,10 @@
             "utemplate/source.py",
             "github:brainelectronics/Micropython-ESP-WiFi-Manager/utemplate/source.py"
         ],
+        [
```

Comment:
As `setup.py` contains `pkg_resources` in the packages definition, it has to be added here although it is a duplicate entry with the deps section